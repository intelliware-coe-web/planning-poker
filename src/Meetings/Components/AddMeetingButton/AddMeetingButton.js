import React from 'react';
import {Link} from 'react-router-dom';

export default function AddMeetingButton() {
    return (
        <Link to="/meeting/add">
            <div className="uk-card uk-card-secondary uk-card-body add-card">
                <h3 className="uk-card-title">+</h3>
            </div>
        </Link>
    );
}