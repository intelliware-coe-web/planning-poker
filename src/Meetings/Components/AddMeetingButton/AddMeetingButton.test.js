import AddMeetingButton from "./AddMeetingButton";

it('AddMeetingButton renders correctly', () => {
    expect(AddMeetingButton()).toMatchSnapshot(); 
});