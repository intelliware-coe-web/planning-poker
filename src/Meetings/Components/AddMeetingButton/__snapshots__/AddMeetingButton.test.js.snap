// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMeetingButton renders correctly 1`] = `
<Link
  to="/meeting/add"
>
  <div
    className="uk-card uk-card-secondary uk-card-body add-card"
  >
    <h3
      className="uk-card-title"
    >
      +
    </h3>
  </div>
</Link>
`;
